src/components/Explorer/index.tsx

```diff
@@ -158,6 +161,7 @@ const Explorer: ComponentProps = ({
         onChange?.({...e, value: e.data});
         setCurrent(e.data);
     }, [onChange]);
+    const rowClass = (data: object) => data?.[keyField] === currentState?.[keyField] && classes.current;
```

Comment:
I think this way is better, as it will not return false as rowClass:
`data?.[keyField] === currentState?.[keyField] ? classes.current : undefined`